### ISSUE TITLE
ionic 5 and Ionicons 5.x Issue: icon name have changed => md-create.svg : 404 Not Found

### ISSUE THREAD

**Issue by cbleu** [eakoriakin/ionic-selectable#246]
Hello,

i have a project that i have migrate to ionic 5 and ionicons 5.

All work nice with ionic-selectable, unless the <ion-icon> to Edit an item that use an a name that is no more used in ionicons 5 !

There is no template to modify that part of the module and i have not found how to change that icon !
So i have a blank icon and a network error 404 Not Found !

Here is the part of the module where the Edit icon is called.

```
<ion-button
  slot="end"
  fill="outline"
  class="ionic-selectable-item-button ios button button-small button-outline button-has-icon-only ion-activatable ion-focusable hydrated"
  ng-reflect-fill="outline"
  ><ion-icon slot="icon-only" name="md-create" ng-reflect-name="md-create" role="img" class="ios hydrated" aria-label="md create"></ion-icon
>
</ion-button>
````
 
![image](https://user-images.githubusercontent.com/878923/76235332-6630e500-6244-11ea-88f4-b3ef95b700cd.png)

Thanks a lot for your amazing module !
Hope you will finish the migration to ionic 5 and ionicons 5 soon ;)

**Comment by edy-ap** [comment]
Thanks for you report, that was resolved in version 4.7.1, is available now on npm

**Comment by jakkie** [comment]
On ionic 6 with ionic-selectable version 4.9.0 I have the exact same issue.
